plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -236,3 +240,22 @@ export function patchCallbackPGPool(
     cb.call(this, err, res, done);
   };
 }
+
+export function patchConnectErrorCallback(
```

Comment:
There is already a function `patchCallbackPGPool` which does almost the same thing. The only difference is the arguments list to the callback which we don't use.
Can we reuse one function for both cases?
Maybe write it with [`arguments` array parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments) instead of listing the different arguments for each case

Reply:
I am not sure about this, think there is some value in keeping descriptive function names. I did rename the function though